portlist/clean.go

```diff
@@ -24,6 +24,8 @@ func argvSubject(argv ...string) string {
 	case ret == "mono" && len(argv) >= 2:
 		// .Net programs execute as `mono actualProgram.exe`.
 		ret = filepath.Base(argv[1])
+	case strings.Contains(ret, " "):
```

Comment:
technically this is unnecessary. strings.Cut is defined to return its input unchanged if it doesn't contain the substring. So this could be after the switch with just `ret, _, _ = strings.Cut(ret, " ")`

Reply:
neat